webpack.embedding-sdk.config.js

```diff
@@ -7,7 +7,6 @@ const webpack = require("webpack");
 const BundleAnalyzerPlugin =
   require("webpack-bundle-analyzer").BundleAnalyzerPlugin;
 const ForkTsCheckerWebpackPlugin = require("fork-ts-checker-webpack-plugin");
-const path = require("path");
```

Comment:
wait... isn't this used before? Why wouldn't our linter catch it?

Reply:
Our `lint-eslint-pure` command seems to run only on `enterprise/frontend frontend e2e` :sad:

Reply:
dang. I think all js, ts files should be no exception 🤔 I mean we don't modify these config files often, but I don't see the reason not to lint them.